core/src/test/scala/integration/kafka/server/DynamicBrokerReconfigurationTest.scala

```diff
@@ -1161,14 +1161,6 @@ class DynamicBrokerReconfigurationTest extends ZooKeeperTestHarness with SaslSet
 
   private def listenerPrefix(name: String): String = new ListenerName(name).configPrefix
 
-  private def configureDynamicKeystoreInZooKeeper(kafkaConfig: KafkaConfig, brokers: Seq[Int], sslProperties: Properties): Unit = {
```

Comment:
@rajinisivaram is it expected that this is unused?

Reply:
@ijuma I had removed the initialization of keystores in the test before starting brokers in an earlier commit, but am using this method in this PR: https://github.com/apache/kafka/pull/4898

Reply:
OK, I will add it back.